src/components/MessageList/ConnectionStatus.tsx

```diff
@@ -1,22 +1,20 @@
 // @ts-check
-import React, { useContext, useEffect, useState } from 'react';
+import React, { FC, useContext, useEffect, useState } from 'react';
+import type { Event } from 'stream-chat';
 
 import { ChatContext, TranslationContext } from '../../context';
-import CustomNotification from './CustomNotification';
+import { CustomNotification } from './CustomNotification';
```

Comment:
can you reorder this import within all the imports up here by depth and then `abc`